packages/communication-react/CHANGELOG.stable.md

```diff
@@ -6,271 +6,106 @@ This log was last generated on Mon, 12 Jun 2023 19:04:18 GMT and should not be m
-  - updated API.md file to remove data-ui-id ([PR #2647](https://github.com/azure/communication-ui-library/pull/2647) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - VideoGallery announces when participants are pinned and unpinned. Added aria labels for pin and unpin menu item ([PR #2662](https://github.com/azure/communication-ui-library/pull/2662) by 79475487+mgamis-msft@users.noreply.github.com)
-  - tee video effects errors to state ([PR #2729](https://github.com/azure/communication-ui-library/pull/2729) by 2684369+JamesBurnside@users.noreply.github.com)
-  - Custom Video Effect tests ([PR #3059](https://github.com/azure/communication-ui-library/pull/3059) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - Introduce E2E tests for Call Composite local tile options ([PR #3096](https://github.com/azure/communication-ui-library/pull/3096) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Remote participant display name used as heading for remote video tile drawer menu. ([PR #2646](https://github.com/azure/communication-ui-library/pull/2646) by 79475487+mgamis-msft@users.noreply.github.com)
-  - Added mapping for inline image e2e support ([PR #2915](https://github.com/azure/communication-ui-library/pull/2915) by 77021369+jimchou-dev@users.noreply.github.com)
-  - Auto-dismiss errors on selection of new effect ([PR #3160](https://github.com/azure/communication-ui-library/pull/3160) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - Implement inline image attachment for chat UI component ([PR #2894](https://github.com/azure/communication-ui-library/pull/2894) by 77021369+jimchou-dev@users.noreply.github.com)
-  - Update storybook to include call readiness public components ([PR #2643](https://github.com/azure/communication-ui-library/pull/2643) by carolinecao@microsoft.com)
-  - Video effect errors test ([PR #3066](https://github.com/azure/communication-ui-library/pull/3066) by 97124699+prabhjot-msft@users.noreply.github.com)
+- Switch video seamlessly from different devices ([PR #2726](https://github.com/azure/communication-ui-library/pull/2726) by jinan@microsoft.com)
+- Center more button in video tile ([PR #2902](https://github.com/azure/communication-ui-library/pull/2902) by edwardlee@microsoft.com)
+- Fix messages thread loading issue when an adapter is updated ([PR #2784](https://github.com/azure/communication-ui-library/pull/2784) by 98852890+vhuseinova-msft@users.noreply.github.com)
+- Addressed the issue where local participant label's luminosity ratio does not meet MAS guidelines ([PR #2795](https://github.com/azure/communication-ui-library/pull/2795) by 109105353+jpeng-ms@users.noreply.github.com)
+- Allow horizontal gallery tiles to resize to allow for better use of space. ([PR #2830](https://github.com/azure/communication-ui-library/pull/2830) by 94866715+dmceachernmsft@users.noreply.github.com)
```

Comment:
nit. Maybe a search of ". (PR" and replace to " (PR" can be done to be consistent and not have periods

Reply:
Updated.